src/extension.ts

```diff
@@ -17,7 +17,7 @@
 
 import * as vscode from 'vscode';
 import * as extensionFunctions from './extensionFunctions';
-import { DidactNodeProvider, TreeNode } from './nodeProvider';
+import { DidactNodeProvider, SimpleNode } from './nodeProvider';
```

Comment:
What is the benefit of SimpleNode over TreeNode?

Reply:
TreeNode is used for Categories & Tutorials. HeadingNode is used for Headings. SimpleNode just gives me something other than vscode.TreeItem to use as a type for all of them.